src/annotator/test/guest-test.coffee

```diff
@@ -238,7 +238,7 @@ describe 'Guest', ->
       guest = null
 
       beforeEach ->
-        sandbox.stub(document, 'title', 'hi')
+        document.title = 'hi'
```

Comment:
This is fine for now. The better solution would be to make `document` injectable into the code under test so it can be easily faked.